github-git/social-open-source.md

```diff
@@ -0,0 +1,158 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst_html_meta:
+  "title": "The social side of open source: Engage, build trust & support contributors"
+  "description lang=en": "A guide to the social elements that will make contributing to open source more impactful, valuable, and successful."
+  "keywords": "GitHub, OpenSource, beginner-friendly"
+  "property=og:locale": "en_US"
+---
+:::{todo}
+Link to relevant sections that cover each of the topics below
+:::
+
+# The social side of open source: Engage, build trust & support contributors  
+
+:::{figure} /images/github/open-source-social-network.png
+:alt: A digital illustration visualizing open-source collaboration on GitHub. At the center, a “Main Repo” connects to contributors performing key actions: forking (branching icon), submitting pull requests (PRs) (merge arrows), reviewing code (magnifying glass over documents), and engaging in discussions (speech bubbles and coding icon). The pyOpenSci logo appears in the bottom right, reinforcing the theme of community-driven contributions.
+
+
+:::
+
+## Why social etiquette matters in open source  
+
+Contributing to open source isn’t just about code—it’s about **collaborating with people you may never meet in real life**. Unlike traditional workplaces, GitHub interactions happen asynchronously, across time zones, and often with volunteers.  
```

Comment:
hell ya

Reply:
@sneakers-the-rat yay! yes my goal with these lessons is to not just teach the technical steps, but also to help people learn that stuff that you and I and others learn "the hard way" by doing it over time. That is part of what I think triggers fear and nervousness. By having our community review all of these lessons, I think collectively, we can pull together our knowledge to make these lessons really great.

Reply:
the goal with this page is to pull together all of the tips we provide on the other pages into a single page as a high level summary.